colcon_ros_gradle/package_identification/ament_gradle.py

```diff
@@ -67,17 +63,22 @@ def gradle_extract_data(build_gradle):
     """
     # Content for dependencies
     content_build_gradle = extract_content(build_gradle)
-    match = re.search(r'apply plugin: ("|\')org.ros2.tools.gradle\1', content_build_gradle)
+    match = re.search(
+        r'apply plugin: ("|\')org.ros2.tools.gradle\1', content_build_gradle)
     if not match:
-        raise RuntimeError("Gradle plugin missing, please add the following to build.gradle: \"apply plugin: 'org.ros2.tools.gradle'\"")
+        raise RuntimeError('Gradle plugin missing, please add the following '
```

Comment:
I can only highly suggest to avoid aligning the following lines. If you happen to change the exception type you will have to reindent the whole block.

Reply:
I will be able to do alignment right sooner or later... :sweat_smile: